kclvm/sema/src/resolver/import.rs

```diff
@@ -98,10 +98,29 @@ impl<'ctx> Resolver<'ctx> {
                             {
                                 match self.ctx.import_names.get_mut(&self.ctx.filename) {
                                     Some(mapping) => {
-                                        mapping.insert(
-                                            import_stmt.name.to_string(),
-                                            import_stmt.path.to_string(),
-                                        );
+                                        if mapping.contains_key(&import_stmt.name.to_string()) {
```

Comment:
It seems that `import_stmt.path` should be used instead of `import_stmt.name` to judge multiple imports, you can modify it here, add comments and tests.

Reply:
fixed. I integrated the reimport check into lint and used `stmt.path` instead of `stmt.name`